docs/assets/style.css

```
@@ -0,0 +1,1279 @@
+  font-size: 1rem;
+  padding-top: 0.75rem;
+  padding-bottom: 0.5rem;
+  margin: 0;
+}
+.tsd-filter-item:not(:last-child) {
+  margin-bottom: 0.5rem;
+}
+.tsd-filter-input {
+  display: flex;
+  width: fit-content;
+  width: -moz-fit-content;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)